examples/cloudflare-workers/src/index.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  ActionGetResponse,
+  ActionPostRequest,
+  ActionPostResponse,
+} from "@solana/actions";
+import {
+  Connection,
+  PublicKey,
+  SystemProgram,
+  TransactionMessage,
+  VersionedTransaction,
+} from "@solana/web3.js";
+import { Hono } from "hono";
+import { cors } from "hono/cors";
+import { Buffer } from "node:buffer";
+
+if (globalThis.Buffer === undefined) {
+  globalThis.Buffer = Buffer;
+}
+
+// you should use a private RPC here
+const connection = new Connection("https://api.mainnet-beta.solana.com");
+
+const app = new Hono();
+
+app.use(cors());
```

Comment:
Does this set the correct CORS headers required for actions and blinks to work? Including with the OPTIONS http method and the `actions.json` route?

Reply:
it does!